CodeFormatCore/src/Format/Analyzer/SemicolonAnalyzer.cpp

```diff
@@ -123,6 +112,10 @@ bool SemicolonAnalyzer::EndsWithSemicolon(LuaSyntaxNode n, const LuaSyntaxTree &
     return token.GetTokenKind(t) == ';';
 }
 
+bool SemicolonAnalyzer::ContainsSemicolon(LuaSyntaxNode n, const LuaSyntaxTree &t) {
+    return n.GetChildTokens(';', t).size() > 0;
```

Comment:
Why not use `n.GetChildToken(';', t).IsToken()`

Reply:
Oh, I wasn't aware of that, changed it to the better alternative. Thank you for explaining, too!